tests/test_crypto_fields.py

```diff
@@ -0,0 +1,132 @@
+import time
+
+from rest_framework import serializers
+from drf_extra_fields.crypto_fields import (
+    CryptoBinaryField,
+    CryptoCharField,
+    _generate_password_key,
+    _encrypt,
+)
+import datetime
+from django.test import TestCase
+from django.conf import settings
+from cryptography.fernet import Fernet, InvalidToken
+from django.utils.translation import gettext_lazy as _
+
+DEFAULT_PASSWORD = b"Non_nobis1solum?nati!sumus"
+DEFAULT_SALT = settings.SECRET_KEY
+
+
+class SaveCrypto(object):
+    def __init__(self, message=None, created=None):
+        self.message = message
+        self.created = created or datetime.datetime.now()
+
+
+class CryptoSerializer(serializers.Serializer):
+    message = CryptoBinaryField(required=False)
```

Comment:
Let's call this `BaseCryptoSerializer` and create other serializers in the test methods using this:

```python
def test_serialization_with_salt(self):
    class CryptoSerializer(BaseCryptoSerializer):
        message = CryptoBinaryField(password=..., salt=...)
```